Makefile

```diff
@@ -4,7 +4,7 @@ develop:
 
 lint:
 	@echo "Linting Python files"
-	flake8 --ignore=E501,E225,E121,E123,E124,E125,E127,E128 wsgiadapter.py
+	python -c "import sys; sys.exit(0 if sys.version_info >= (2,7) else 1)" && flake8 --ignore=E501,E225,E121,E123,E124,E125,E127,E128 wsgiadapter.py || echo "not linting on python 2.6"
```

Comment:
Yes, flake8 is no longer compatible. I've added a comment besides from the commit message